src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -208,6 +210,24 @@ public boolean equals(Object o)
     User getUserById(String id);
 
     /**
+     * Gets all {@link Guild}s that contain all given users as their members.
```

Comment:
This has to be a full qualified path to the `Guild` object like `{@link net.dv8tion.jda.core.entities.Guild Guild}`
Always include the path and an alias name.